### PR TITLE
feat(icon): update info icon

### PR DESCRIPTION
**Pull request by mupkoo**
### Description

Update the `Info` icons to the new ones from **Product Iconography**. This change comes from the design team and it is related to [Talent application form redesign (2021)](https://toptal-core.atlassian.net/browse/TACO-1173). 

You can see the design here:
https://www.figma.com/file/Lp9meoy0qlvH05GNUeQ2lW/Talent-Application-Form?node-id=1608%3A21041

You can find the new icons here:
https://www.figma.com/file/TqaGgbpjGSUDf7qq153Isq/Product-Iconography?node-id=1%3A1408

You can checkout this Slack thread for more context as well:
https://toptal-core.slack.com/archives/C02R32BHD3K/p1644585756168799

### How to test

- FIXME: Add the steps describing how to verify your changes

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Shot 2022-02-15 at 16 14 30 | Shot 2022-02-15 at 16 14 10 |
| Shot 2022-02-15 at 16 14 50 | Shot 2022-02-15 at 16 15 00 |
| Shot 2022-02-15 at 16 15 40 | Shot 2022-02-15 at 16 15 24 |

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- n/a ~~Annotate all `props` in component with documentation~~
- n/a ~~Create `examples` for component~~
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Comment by mupkoo** [comment]
@toptal-bot run all

**Review by TomasSlama — COMMENTED** [review]
LGTM 👍, I'm just wondering why this screen is changed image `__image_snapshots__/notifications-stream-options-effect-2-snap.png`

File: .changeset/silent-mice-brake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': patch
+---
+
+Update `Info` to the new one from Product Iconography
```
Comment (TomasSlama) [inline .changeset/silent-mice-brake.md:5]:
```suggestion
### Icon

Update `Info` to the new one from Product Iconography
```

**Review by TomasSlama — APPROVED** [review]

**Review by separatio — APPROVED** [review]
LGTM. 👍

File: .changeset/silent-mice-brake.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@toptal/picasso': patch
+---
+
+### Icon
```
Comment (separatio) [inline .changeset/silent-mice-brake.md:5]: Forgot to check the changeset.
I'd change this to `Info Icon`.

**Comment by TomasSlama** [comment]
@mupkoo commit message Update the wording of the changeset, needs to be changed to something like `chore: update the wording..`

**Pull request merged**